src/assets/styles/20-tools/_extends/_images.scss

```diff
@@ -12,6 +12,7 @@
  */
 %img-responsive {
   > img {
+    background-color: var(--colour-amber-05);
```

Comment:
This adds a background color to wherever this extend is used (Card, Author, ImageCard, FeaturedPromo). I think this is a useful bit of UI repetition as it means wherever an image fails to load, a consistent bg color will be shown which will become a common pattern for users to recognise and understand.